Weekly_Work/Week1/Code_changes-git/Metrics_Root_class.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
```

Comment:
Should be python3

Reply:
That line is generated when I download the jupyter notebook as a python script. At this point in time, I am focusing mainly on the structure and functionality so I am ignoring things like docstrings and comments in this pull request. In the next pull request, I will add those, complete with the changes required.
Will make that python3.

Reply:
Good. No problem.